lib/partials.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = function (grunt, options, wrapData) {
+  var wrapTpl = grunt.file.read(wrapData.partialPath);
+  var files = options.files;
+  var content;
+  var path;
+  wrapTpl = wrapTpl.split('<!-- CONTENT -->');
+  wrapData.cache = {};
+  wrapData.files = [];
+  wrapData.lineOffset = wrapTpl[0].split(/\n/).length - 1;
+  for (var i = 0; i < files.length; i += 1) {
```

Comment:
Any reason not to use `forEach` or `map` (assigning return value to `wrapData.files`)?